widgets/cu-xmpp-chat/lib/index.css

```diff
@@ -343,13 +343,16 @@
         color: #909090;
         cursor: pointer; }
     .cse-chat .chat-frame .chat-tab-content.chat-users {
+      overflow-x: hidden;
       overflow-y: auto;
       flex-grow: 1;
       margin: 0.5em 0; }
       .cse-chat .chat-frame .chat-tab-content.chat-users .chat-info-user {
         color: white;
         padding: 0.5em;
-        cursor: pointer; }
+        cursor: pointer;
+        text-overflow: ellipsis;
+        overflow-x: hidden; }
```

Comment:
Move the brace to a new line and aligned with the opener (yes I know it was wrong before your checkin).